website/docs/installation/documentdb/deb.md

```diff
@@ -4,6 +4,15 @@ sidebar_position: 2
 
 # DEB package
 
+:::note
+We provide different `.deb` packages for various deployments.
+
+- For most use cases, we recommend using the production package (e.g., `documentdb.deb`).
+- For debugging purposes, use the development package (contains either `-dev` or `-dbgsym` suffix e.g., `documentdb-dev.deb`/`documentdb-dbgsym.deb`).
+  It includes features that significantly slow performance and is not recommended for production use.
+
+:::
+
 FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
 
 Download the DocumentDB `.deb` package from [our release page](https://github.com/FerretDB/documentdb/releases/),
```

Comment:
For example, it would look natural to describe the content of that page (what kinds of packages are there) after we gave a link to that page